tests/std/tests/P2374R4_checked_arithmetic_operations/test.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cstdint>
+#include <limits>
+#include <ranges>
+#include <utility>
+
+using namespace std;
+
+// Check MSVC-STL internal machinery
+
+template <class Int>

```

Comment:
It's dangerous for casing to be the only difference between identifiers (or an identifier and a keyword) when confusion would be possible. That is, `class Class` would read weirdly but isn't especially dangerous because accidentally saying `class` instead of `Class` wouldn't compile. However, accidentally saying `int` instead of `Int` would be silently damaging here. I strongly believe that this should be `Integer` (or anything similar).

Reply:
Really just `T` would be fine.